IRremoteESP8266.cpp

```diff
@@ -450,6 +450,45 @@ void ICACHE_FLASH_ATTR IRsend::sendDenon (unsigned long data,  int nbits) {
   ledOff();
 }
 
+// Gree, from https://github.com/ToniA/arduino-heatpumpir/blob/master/GreeHeatpumpIR.cpp
+void ICACHE_FLASH_ATTR IRsend::sendGree(unsigned char data[]) {
```

Comment:
I don't really have an opinion as long as it works for my purposes, which is to be able to use https://github.com/Daniel-t/esp8266IRServer (or rather, my fork https://github.com/scop/esp8266IRServer/tree/gree) with the Gree protocol.

I suppose an unsigned long, int nbits method signature would be nice if it could be done like that, so that it would be similar to most of the other send* methods and integrate more seamlessly with the esp8266IRServer's way of handling args.

For the way I currently do it, see my above esp8266IRServer gree branch, and the gree.js javascript file there, as well as how the gree case gets handled in irServer.ino.

But I think I'll need some hand holding in order to be able to implement this, and it might take me a while.